src/Plugins/ITKImageProcessing/tools/itk_filter_generator.py

```diff
@@ -718,7 +718,7 @@ def get_itk_filter_struct(filter_data: FilterData) -> str:
   auto createFilter() const
   {{
     using FilterType = {filter_type};
-    auto filter = FilterType::New();{setters_str}
+    auto nx_filter = FilterType::New();{setters_str}
```

Comment:
```suggestion
    auto filter = FilterType::New();{setters_str}
```
Looks like this accidentally got caught by the renaming.